README.md

```diff
@@ -5,13 +5,13 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+Tutor's Pet is a student management application that aims to reduce the workload of tutors. It provides a one stop
+ platform for tutors to handle all of their administrative matters.
+- Tutor's Pet allows you to keep track of all your classes, students and lessons.
+- Tutor's Pet allows you to mark students' attendance and participation for a particular lesson.
```

Comment:
Not sure if this would sound more in line with the other points since the word 'your' was used in both the other.
```suggestion
- Tutor's Pet allows you to mark your students' attendance and participation for each lesson.
```